spring-cloud-azure-samples/azure-appconfiguration-conversion-sample/initial/src/main/java/sample/convert/User.java

```diff
@@ -0,0 +1,95 @@
+/**
```

Comment:
`maven-checkstyle-plugin` errored out due to the second * on this line. Removing it resolved the error.